spikewrap/examples/load_data.py

```diff
@@ -0,0 +1,24 @@
+from pathlib import Path
+
+from spikewrap.pipeline.load_data import load_data
+
+base_path = Path(
+    # r"/ceph/neuroinformatics/neuroinformatics/scratch/jziminski/ephys/test_data/steve_multi_run/1119617/time-short-multises"
+    r"X:\neuroinformatics\scratch\jziminski\ephys\test_data\steve_multi_run\1119617\time-short-multises"
+)
+
+sub_name = "sub-1119617"
+sessions_and_runs = {
+    "ses-001": [
+        "run-001_1119617_LSE1_shank12_g0",
+        "run-002_made_up_g0",
+    ],
+    "ses-002": [
+        "run-001_1119617_pretest1_shank12_g0",
+    ],
+    "ses-003": [
+        "run-002_1119617_pretest1_shank12_g0",
+    ],
+}
+
```

Comment:
How many dataset do you think a person will like to load simultaneously?
I think it is cool to save and read files maintaining the specs we have decided upon, however I think they should not be the keys of the data dict, because they are long, hard to read and typos are hard to spot, potentially leading to further confusions.
Previously I made simple classes with attributes `ses` `subjects` to hold the data information. But this does not have to be the case. We can discuss about it if you like to change this implementation.

Reply:
We discussed this issue (dict keys #123) and general validation across projects that it would be nice to align (#124).